hoot-js/src/main/cpp/hoot/js/conflate/js/ScriptMatchCreator.cpp

```diff
@@ -117,14 +117,6 @@ class ScriptMatchVisitor : public ConstElementVisitor
     {
       _getSearchRadius = Persistent<Function>::New(Handle<Function>::Cast(value));
     }
-
-    boost::function<bool (ConstElementPtr e)> f =
```

Comment:
Did this logic get moved or removed completely?  Was looking for it elsewhere and didn't see it.